### DISCUSSION TITLE
Adding a new provider next to AWS, GCP and Azure

### DISCUSSION THREAD

**Discussion by louwersj** [aquasecurity/tfsec#721] [Developement]
Should we have some level of guidance / process description on how the community could start adding a new cloud. I would be very interested to add checks for an additional terraform provider so we can extend the tfsec checks. 

At current AWS, GCP and Azure are present,.... I would have a need for Oracle Cloud (OCI) checks and love to start building them and add them to the tfsec codebase. 

Guidance on the best way of building this extension would be very welcome to ensure we start in the right fashion.

**Comment by liamg** [comment]
We recently reworked our contributing guide here: https://github.com/aquasecurity/tfsec/blob/master/CONTRIBUTING.md

If there's anything else you need, please ask and we can add to the docs :+1:
